### PR TITLE
Fine tune default tour

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

**Comment by mmiladi** [comment]
I need to wait for docker build to test the update. So please don't merge it now.

**Comment by bgruening** [comment]
Ready? Looks good to me!

**Comment by mmiladi** [comment]
Let me run this docker image tag first ;-)

File: tours/graphclust_tutorial.yaml [hunk 1]
```diff
@@ -2,23 +2,38 @@ name: GraphClust workflow
...
                  </b>
-                 </dir>"
+                 </lo>"
       backdrop: true
 
+    - title: "<b>Log in</b>"
+      element: '#user > li > a'
+      intro: " To be able to use workflows you should be logged in. So if you already have an account
+              simply <b>log in</b> or otherwise <b>register</b> by clicking on <b>'User'</b>.<br>
+              Within a <b>Docker</b> Galaxy-GraphClust everyone can register on default.
```
Comment (bgruening) [inline tours/graphclust_tutorial.yaml]: s/on/by/

File: tours/graphclust_tutorial.yaml [hunk 2]
```diff
@@ -97,42 +117,48 @@ steps:
...
+        
+                   <li> <b>CLUSTER</b>: cluster number</li>
+                   <li> cm_score: Covariance model bit score indicates how well sequence matches to the CM 
+                   model of the cluster.</li>
+                   <li> sequence origin (whether it orginnates from dense center <b>MODEL</b> 
+                   or from Infernal <b>CMSEARCH</b> or preclustering <b>CDHIT</b>)</li>
+                   <li>Input fasta sequence id seperated into <b>ORIGID</b> and <b>ORIGHEAD</b> sections</li>
                  </b>
-                 </dir>"
+                 </lo>"
```
Comment (bgruening) [inline tours/graphclust_tutorial.yaml]: ol? or lo'l? ;)

Comment (mmiladi) [reply tours/graphclust_tutorial.yaml]: :))

File: tours/graphclust_tutorial.yaml [hunk 3]
```diff
@@ -2,23 +2,38 @@ name: GraphClust workflow
 description: Simple instructions for using GraphClust workflow for clustering RNA sequences
 title_default: "<b>GraphClust</b>"
 steps:
-    - title: "<b>A tutorial on GraphClust(Clustering RNA sequences)</b>"
+    - title: "<b>A tutorial on Galaxy-GraphClust(Clustering RNA sequences)</b>"
       content: "This tour will walk you through the process of <b>GraphClust</b> to cluster RNA sequences.<br><br>
-                Read and Follow the instructions before clicking <b>'Next'</b>.<br><br>
+                In the forthcoming windows please Read and Follow the instructions before clicking <b>'Next'</b>.<br><br>
```
Comment (bgruening) [inline tours/graphclust_tutorial.yaml]: read and follow, capital vs. non-capital?

**Pull request merged**